### PR TITLE
Crateria Crystal Flash strats

### PR DESCRIPTION
**Pull request by blkerby**

### CODE REVIEW

**Review by kjbranch — COMMENTED** [review]
Looks like youre missing some rooms, is it worth having in case you can't reset the room?  flyway, pre-map flyway, final missile, bowling alley path, crab maze, crateria kihunter, kago room, lower mushrooms, terminator

**Comment by kjbranch** [comment]
This may look like kind of an odd PR, because of some of the simple strats that are missing, so I'll give some context behind it. Because of some complicated reasons, Map Rando is not currently able to use CF strats which move between nodes (they must be on a link to itself). The inclusion of them could currently have some negative unintended consequences. So the strats in this PR are intended to be simple but fairly thorough, but not so much so as to be adding new obstacles to rooms, etc.

File: region/crateria/central/Climb.json [hunk 1]
```diff
@@ -258,6 +259,11 @@
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "Crystal Flash",
+      "requires": ["h_canCrystalFlash"]
```
[the remark this answers is not in the document]
Comment (blkerby) [reply region/crateria/central/Climb.json]: Done, put it in node 6 now.

File: region/crateria/central/Crateria Super Room.json [hunk 2]
```diff
@@ -670,6 +675,11 @@
         }
       }
     },
+    {
+      "link": [2, 2],
+      "name": "Crystal Flash",
+      "requires": ["h_canCrystalFlash"]
+    },
```
[the remark this answers is not in the document]
Comment (blkerby) [reply region/crateria/central/Crateria Super Room.json:682]: Good catch, that's an important one. Done.

File: region/crateria/east/East Ocean.json [hunk 3]
```diff
@@ -400,6 +405,11 @@
         }
       }
     },
+    {
+      "link": [2, 2],
+      "name": "Crystal Flash",
```
[the remark this answers is not in the document]
Comment (blkerby) [reply region/crateria/east/East Ocean.json:410]: Done.

**Review by kjbranch — APPROVED** [review]

**Pull request merged**